Deprecating project

colincasey opened the pull request:
The types that were defined in this repository now live at [github.com/forcedotcom/sf-fx-runtime-nodejs](https://github.com/forcedotcom/sf-fx-runtime-nodejs)

[GUS-W-11944771](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000019Rqb8YAC/view)

colincasey committed:
Deprecating project

The types that were defined in this repository now live at [github.com/forcedotcom/sf-fx-runtime-nodejs](https://github.com/forcedotcom/sf-fx-runtime-nodejs)

[GUS-W-11944771](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000019Rqb8YAC/view)

joshwlewis reviewed (approved):
Seems OK. I guess we can use `npm deprecate` at some point in the future when we want to push folks to use `sf-fx-runtime-nodejs` instead.

colincasey committed:
Merge branch 'main' into deprecate_repository

colincasey committed:
Deprecating project

The types that were defined in this repository now live at [github.com/forcedotcom/sf-fx-runtime-nodejs](https://github.com/forcedotcom/sf-fx-runtime-nodejs)

[GUS-W-11944771](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000019Rqb8YAC/view)

colincasey commented:
waiting on https://github.com/forcedotcom/sf-fx-runtime-nodejs/pull/435
